web-client/src/presenter/computeds/AdvancedSearch/practitionerSearchHelper.ts

```diff
@@ -78,10 +80,11 @@ export const practitionerSearchHelper = (
     result = {
       ...result,
       formattedSearchResults: paginatedResults,
-      numberOfResults: Number(formatPositiveNumber(searchResults.total)),
+      numberOfResults: formatPositiveNumber(searchResults.total),
```

Comment:
this is still going to give nan which is why we set it to 0 before. I get the idea here and think this is better though.